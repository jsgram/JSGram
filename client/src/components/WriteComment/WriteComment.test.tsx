import React from 'react';
import ConnectedWriteComment, { WriteComment } from './index';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

describe('WriteComment smart component', () => {
    let renderer;
    const mockStore = configureStore();
    const props = {
        userId: 'somevalue',
        postId: 'somevalue',
    };
    const initialState = {
        loggedId: 'somevalue',
        loggedUsername: 'somevalue',
        onChangeComments: jest.fn(() => 'somevalue'),
    }

    beforeEach(() => {
        const store = mockStore(initialState);
        renderer = shallow(<Provider store={store}><ConnectedWriteComment {...props} /></Provider>);
    });

    test('render-success', () => {
        expect(renderer.find(ConnectedWriteComment).length).toEqual(1);
    });
});
