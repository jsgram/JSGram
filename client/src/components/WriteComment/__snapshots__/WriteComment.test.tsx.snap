// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WriteComment smart component render - success 1`] = `ShallowWrapper {}`;
